sunpy/net/helioviewer.py

```diff
@@ -51,7 +52,8 @@ def data_sources(self):
                             data_sources_dict[(name, inst, None, wavelength)] = params['sourceId']
                         else:
                             for wave, adict in params.items():
-                                data_sources_dict[(name, inst, wavelength, wave)] = adict['sourceId']
+                                data_sources_dict[(name, inst, wavelength, wave)
+                                                  ] = adict['sourceId']
```

Comment:
ewww really?

Whatever.

Reply:
It's what you get for having a stupid line length.